package io.github.kxng0109.taskflow.exception.dto;

public record ErrorResponse(String message){
}
